src/Arcus.Security.Providers.AzureKeyVault/Configuration/SecretProviderConfigurationBuilderExtensions.cs

```diff
@@ -1,12 +1,15 @@
-﻿using Arcus.Security.Core;
+﻿using System;
+using Arcus.Security.Core;
 using GuardNet;
 using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
 
 namespace Arcus.Security.Providers.AzureKeyVault.Configuration
 {
     /// <summary>
     /// Provide extensions to use an <see cref="ISecretProvider"/> implementation to retrieve secret values from configuration tokens.
     /// </summary>
+    [Obsolete("In favor of the secret store concept, use " + nameof(SecretStoreBuilderExtensions.AddAzureKeyVault) + " instead during the " + nameof(IHostBuilderExtensions.ConfigureSecretStore))]
```

Comment:
This isn't really deprecated imo; if people want to do it themselves they still can, no?

Reply:
Hmm, leaving this 'un-deprecated' would mean to me that you should/could also use your secrets in the `IConfiguration` instead. Making this obsolete and pointing to the "right" functionality gives some guidance here I think is for the benefit for the consumers, no?

Reply:
It gives guidance but also means that we are cutting support for it and I'm not sure if we want to; otherwise they will go back to the official one.

So I would wait until our secret store is out and can still deprecate it later on imo, sounds fair?

Reply:
Ok, yes.